how-to-guides/present-cross-validation-results/scripts/neptune_cross_validation.py

```diff
@@ -0,0 +1,110 @@
+# How to present CV with Neptune
+import neptune.new as neptune
+import torch
```

Comment:
Pls use use black formatting in both colab and script.

Reply:
Done ✅